src/components/Navigation/Navigation.js

```diff
@@ -82,7 +85,8 @@ const Navigation = (props) => {
             <List className="list">
                 {NavList.map((NavItem) => (
                     <ListItem
-                        button
+                    // button prop empty? What is that?
```

Comment:
similar comment as above. React JSX syntax makes it so that:
`<ListItem button/>` 
is the same as 
`<ListItem button="true"/>`

https://v3.material-ui.com/api/list-item/